packages/@aws-cdk/aws-batch/lib/job-definition.ts

```diff
@@ -458,6 +523,9 @@ export class JobDefinition extends Resource implements IJobDefinition {
       user: container.user,
       vcpus: container.vcpus || 1,
       volumes: container.volumes,
+      fargatePlatformConfiguration: container.fargatePlatformConfiguration ? {
```

Comment:
Sorry, my github-fu is weak. Not sure how to put a comment on code that isn't a part of the PR.

But above, when setting the `resourceRequirements`, don't we need a way to pass those in for fargate?
```
resourceRequirements: container.gpuCount
  ? [{ type: 'GPU', value: String(container.gpuCount) }]
  : undefined
```

Also, pretty sure that `vcpus` needs to be not set